apps/web/lib/services/blockchain/__tests__/integration.test.ts

```
@@ -0,0 +1,766 @@
+/**
+ * Integration tests for Mock Blockchain Services
+ * Tests end-to-end user journeys, service dependencies, and cross-service interactions
+ */
+
+import { describe, test, expect, beforeEach } from 'bun:test';
+import { MockBlockchainServiceFactory } from '../mock-service-factory';
+import { BadgeType, NFTError, BadgeError, ProgressError } from '~/lib/types/blockchain/contract-interfaces.types';
```

Biome (2.1.2)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)